src/containers/guest-home-page/login-form/LoginForm.jsx

```diff
@@ -32,6 +32,10 @@ const LoginForm = ({
     openModal({ component: <ForgotPassword /> })
   }
 
+  const buttonValidate = (emailValue) => {
+    return emailValue?.length && data?.password?.length ? false : true
```

Comment:
Look at comment from sonar 
![image](https://github.com/Space2Study-UA-1156/Client-02/assets/67462207/a80a535a-ac1e-407a-8765-9e5f87f0f070)
You can write smth like
```suggestion
    return !(emailValue?.length && data?.password?.length)
```